dedis-cothority.toml

```diff
@@ -1,39 +1,40 @@
 [[servers]]
   Address = "tcp://78.46.227.60:7770"
-  Public = "5eI+WFOaCdMhHY+g+zR11IZV4MBtg+k8jm59FqqHwQY="
+  Public = "e5e23e58539a09d3211d8fa0fb3475d48655e0c06d83e93c8e6e7d16aa87c106"
```

Comment:
But this whole file is End Of Life, right? Don't we expect to manage this via CISC now?

Reply:
yes, soon... but I think we still should have a list of more reliable nodes. So I should perhaps clean this up.